### ISSUE TITLE
MVP Client Relayer: Periodic Client Updates

### ISSUE THREAD

**Issue by ebuchman** [informalsystems/ibc-rs#216]
MVP of a relayer should be something that is configured for two existing chains, with clients already existing on them, and just relays ClientUpdate datagrams back and forth. We should target two versions of this: local chain and cosmos sdk chain.

## Local Chain

It may be best to start by integrating against a local chain (#158) using using our own ics02/07 handlers (#159) and tendermint light client (#90). This shouldn't require any signing or block on any dependencies (can use v0.16 series of tendermint-rs).

This will allow in process testing with minimal dependencies and shouldnt require waiting for tendermint-rs to update to tendermint v0.34. 

-> This is further decomposed and tracked in #381

## Cosmos SDK Chain

Then to get it working against a real chain will require updating to the latest tendermint-rs post compatibility with Tendermint v0.34 and will target the latest version of the Cosmos SDK (still unreleased Stargate). Signing will depend on #47 and should just go with whatever is simplest, though note the relayer key's address will need to have an account on both chains. This will also involve updating to the new tendermint websocket API and using that for event subscriptions.

Merkle proofs can be ignored by the relayer for now - in the MVP we can just trust the full nodes.

**Comment by ancazamfir** [comment]
At this point we have basic relayer functionality running. But that does not include updating clients in the absence of transactions requiring client updates. I believe it is still useful to have some test command that just updates clients on every `NewBlock` event. This is a way to stress the relayer and chains and also to run in parallel with the V1 version.

I am changing the milestone for this to V1.

**Comment by adizere** [comment]
Duplicate of #786, closing in favour of that.
